tools/ansible/README.md

```diff
@@ -54,6 +54,10 @@ To use WSU which builds WMCB for development purposes, set value of `build_wmcb`
 ```
 $ ansible-playbook -i hosts tasks/wsu/main.yaml -v -e "{build_wmcb: True}"
 ```
+
+**API rate limit exceeded error when running WSU:** \
+You might encounter Api rate limit exceeded error while running WSU playbook. The issue occurs due to github rate-limiting unauthenticated requests at 60 requests per hour. At the moment there is no fix for this issue. In order to run the Ansible playbook successfully, wait for the rate-limit to reset (at most 1 hour).
```

Comment:
- Please use `API`. 
- Please mention that the GitHub API is used in the WSU playboook and how we are using it. 
- Rather than saying there is no fix, please say the workaround is to wait.